tests/PHPStan/Analyser/data/method-phpDocs-inheritdoc.php

```diff
@@ -69,4 +69,12 @@ private function privateMethodWithPhpDoc()
 
 	}
 
+	/**
```

Comment:
Can you please add another method that looks like doFoo() and has @inheritdoc above it? It should contain second die; which will NodeScopeResolverTest cause to assert that everything works with both options. (Also make sure that assertTypes is called with useCache=false).